packages/loader/container-loader/src/container.ts

```diff
@@ -170,15 +171,21 @@ export enum ConnectionState {
  */
 export async function waitContainerToCatchUp(container: IContainer) {
     // Make sure we stop waiting if container is closed.
-    const containerClosedMessage = "Container is closed";
     if (container.closed) {
-        throw new Error(containerClosedMessage);
+        throw new UsageError("Container closed");
```

Comment:
```suggestion
        throw new UsageError("waitContainerToCatchUp: Container closed");
```

I know it's in the callstack, but stacks are sometimes woefully difficult to use, so pragmatically I tend towards really explicit messages.